src/mmap_windows.rs

```diff
@@ -219,8 +221,20 @@ impl Drop for MmapRegion {
     fn drop(&mut self) {
         // This is safe because we mmap the area at addr ourselves, and nobody
         // else is holding a reference to it.
+        // NOTE(lpetrut): The size must be set to 0 when using MEM_RELEASE,
```

Comment:
No need to add your name here. :)

Reply:
Sure, sorry about that. It's just a habit from working on other projects that requested notes, TODOs, etc to include the name/nickname of the one adding those.